CONTRIBUTORS.md

```diff
@@ -7,26 +7,34 @@ Chapel release:
 Contributors to the current release
 -----------------------------------
 * Ben Albrecht, [Cray Inc.]
-* Sidhant Bendre, individual contributor
+* Himanshu Bhadani, individual contributor
+* Akshansh Bhanjana, individual contributor
 * Paul Cassella, [Cray Inc.]
 * [Brad Chamberlain], [Cray Inc.]
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology Roorkee]
+* Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, Sri City] (former individual contributor)
 * Lydia Duncan, [Cray Inc.]
 * Michael Ferguson, [Cray Inc.]
 * Ben Harshbarger, [Cray Inc.]
 * Jenna Hoole, [Cray Inc.]
```

Comment:
I am "Jenna Hoole Starkey" (or just "Jenna Starkey" if we don't like middle names) these days.